protocols/v2/binary-sv2/serde-sv2/src/primitives/sequences/option.rs

```diff
@@ -242,7 +262,14 @@ impl<'s> Sv2Option<'s, u32> {
                 data: Some(inner),
             }
         } else {
-            panic!()
+            // this is an already valid seq should be safe to call the unwraps.
```

Comment:
why is this unwrap safe?

Reply:
Yes because if `self.data` is `None` then `self.seq` is `Some(Inner)` (it is the same as above!)